### ISSUE TITLE
Add "Go to Commit Message" keyboard shortcut

### ISSUE THREAD

**Issue by mdiep** [desktop/desktop#1719]
### Description
Cmd-Shift-C, to bring the commit message textfield into focus, is probably my most-used shortcut in the old Desktop. I'd love to have it in new Desktop.

### Version
**GitHub Desktop version:** 0.5.3
**OS version:** macOS 10.12.5

### Steps to Reproduce
1. Select file in Changes list
2. Press Cmd-Shift-C (or something else)

**Expected behaviour:** Bring commit message text field into focus

**Actual behaviour:** Nothing

**Comment by joshaber** [comment]
I used it a lot in Classic too, but our keyboard navigation is generally better in TNG. So once I broke my muscle memory, I haven't missed it.

**Comment by jspahrsummers** [comment]
@joshaber I'm guessing your system-wide preference is set so that Tab only moves between text fields (because it's the default). When it's set so that Tab moves between everything:

<img width="780" alt="screen shot 2017-06-06 at 22 10 14" src="https://user-images.githubusercontent.com/432536/26852112-12cdfcda-4b05-11e7-8e64-d96d5fc9cfd5.png">


… the lack of a shortcut like this gets really painful.

**Comment by bruncun** [comment]
@joshaber Could you please elaborate on how keyboard navigation has improved? Was this shortcut removed by design? I'm finding myself tabbing tediously, and the user guides do not cover keyboard navigation.

**Comment by davesmiths** [comment]
I miss this keyboard shortcut as well.

I can see the keyboard navigation is much improved as I can tab to most things now where I couldn't before. However, I would still appreciate a shortcut to the Summary field as to me it's as used as Pushing (Cmd+P) or beginning the switch to a different repository (Cmd+T).

**Comment by BBonifield** [comment]
Has there been any additional thought on this one?

**Comment by billygriffin** [comment]
Adding the help wanted label. We don't think this is going to be used by a large portion of users, but we're absolutely open to the original suggestion here if someone from the community wants to implement it. We agree that it would likely be useful to power users.

**Comment by tierninho** [comment]
For any open-source contributor looking to work on this, a good place to start is below.

There was a similar feature, which implemented a new keyboard shortcut:
https://github.com/desktop/desktop/pull/5822/files

Please reach out to the team with any questions. Thanks!

**Comment by billygriffin** [comment]
@tierninho This should be closed when #5822 lands, so others shouldn't pick this up. Thanks for providing more context for folks though!

**Comment by rinogo** [comment]
tldr; Try `⌘G`.

Bonus: You can then use `⌘<Enter>` to submit the commit and then `⌘P` to Push.

**Comment by rinogo** [comment]
Hahahah I forgot how to do this, searched, and learned from my own answer. 🤪🤷‍♂️

**Comment by rinogo** [comment]
Late 2021 🤦‍♂️ - Hi again 👋
